### PR TITLE
Expose plutus-enabled GHC from the flake.

### PR DESCRIPTION
**Pull request by shlevy**
Enables the bitte deployment to use master.

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -111,6 +111,15 @@
           plutus-playground-server = topLevel.plutus-playground.server;
           marlowe-website = topLevel.marlowe-web;
           web-ghc-server = plutus.haskell.project.hsPkgs.web-ghc.components.exes.web-ghc-server;
+          # must be in $PATH of web-ghc-server
+          ghcWithPlutus = plutus.haskell.project.ghcWithPackages (ps: [
```
Comment (michaelpj) [inline flake.nix:115]: Shouldn't we just expose a wrapped `web-ghc`? We have such a thing, it's here: https://github.com/input-output-hk/plutus/blob/master/nix/pkgs/default.nix#L164

Comment (shlevy) [reply flake.nix:115]: @michaelpj I think we need this for marlowe-playground as well, which currently doesn't use webghc. I'll update the comment and make a note to switch to the wrapped webghc.

Comment (michaelpj) [reply flake.nix:115]: Well, we have some nixos modules lying around for the old deployment that AFAIK do this stuff right, so we should be able to borrow from that. That's where I found this web-ghc expression!

**Review by michaelpj — APPROVED** [review]
Happy to merge so long as you're checking out what's going on with the wrapped versions. I do think having wrapped versions is better!

**Comment by shlevy** [comment]
Yeah I admit I'm a bit confused as to how the nixos deployment works (I don't see any wrapping of Marlowe nor other PATH manipulation) but since we're going to update marlowe-playground to use webghc anyway we can do it then.

File: flake.nix (hunk 1 above)
Comment (shlevy) [reply flake.nix:115]: Will be fixed vy https://github.com/input-output-hk/plutus/pull/3739

**Pull request merged**